front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -301,6 +302,8 @@ export default function AssistantBuilder({
       actionMode = "DUST_APP_RUN";
     } else if (isTablesQueryConfiguration(action)) {
       actionMode = "TABLES_QUERY";
+    } else if (isProcessConfiguration(action)) {
+      // not supported in the builder yet
```

Comment:
I think it's useful to clearly say it's unsupported yet but happy to remove if you don't see the point.